test/javascript/components/mentoring/mentor_inbox.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
```

Comment:
@ErikSchierboom @SleeplessByte I've rewritten the test in this way in order to not test implementation details. Let me know what you think.

This also brings me to a bigger discussion as to whether we want to move to tests like these when dealing with JS components. The Rails system tests that we have right now are OK, but I'm now seeing an overlap between these JS tests and Rails system tests. I think the Rails system tests should just check that the components are rendered correctly (which tests that the correct files have been required by webpacker, etc),  but not the component's more specific behavior such as pagination.

The only good thing that I see with Rails system tests is that we could walk through them in our browser and it allows other developers to skin it much easier. But, we could do that by using something like [react-storybook](https://storybook.js.org/)

Reply:
Yeah, if that is the end goal, storybook is a good choice

Reply:
I like it!